modules/ocf_backups/files/backup-mysql

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash -eu
-# Creates a MySQL backup in the current directory. Intended to be called by
-# rsnapshot.
-set -o pipefail
-mysqldump \
-    --defaults-file=/opt/share/backups/my.cnf \
-    --events \
-    --triggers \
-    --all-databases --single-transaction | \
-    pigz > "mysql-$(date +%F).sql.gz"
+# Creates a MySQL backup in the current directory, with one file per database.
+# Intended to be called by rsnapshot.
+
+# credit:
+# https://mensfeld.pl/2013/04/backup-mysql-dump-all-your-mysql-databases-in-separate-files/
+
+set -euo pipefail
+
+databases=$(mysql --defaults-file=/opt/share/backups/my.cnf -e "SHOW databases;" | \
+		   grep -Ev "(Database|information_schema|performance_schema)")
```

Comment:
I mentioned yesterday that I thought anchoring this regex would be safer in case a username or something happens to contain `Database` or something, but given that the output here has a table format and everything, I think just leaving it like this would be the easiest and should be safe enough.